codespell_lib/tests/test_basic.py

```diff
@@ -198,7 +198,7 @@ def test_custom_regex(tmpdir, capsys):
     """Test custom word regex."""
     d = str(tmpdir)
     with open(op.join(d, 'bad.txt'), 'w') as f:
-        f.write('abandonned_abondon\n')
+        f.write('abandonned-abondon\n')
```

Comment:
I just kept the existing test working with a dash rather than an underscore. I could add a test specifically for bringing back underscores as per the addition to README.rst